modules/ROOT/pages/access-management-dns.adoc

```diff
@@ -3,12 +3,18 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-If this step was not performed prior to installation, you must provide the DNS or IP address where your Anypoint Platform Private Cloud Edition installation is exposed to the local network. This is required for Mule to communicate with Anypoint Platform.
+If you did not perform this step before installation, you must provide the DNS or IP address where your Anypoint Platform Private Cloud Edition installation is exposed to the local network. This is required for Mule to communicate with Anypoint Platform.
 
 . From Anypoint Platform, click Access Management.
 . Click the *DNS/IP* tab.
 . In the *Platform DNS/IP* field, enter the address where you are hosting the platform.
 . Click *Save*.
 
 
-image::on-premises-features-2f348.png[]
+[NOTE]
+==
+If you perform a DNS update that changes the endpoints Mule runtimes use to connect to Runtime Manager, you must restart the Mule runtimes before they can connect to the new endpoints.
```

Comment:
```suggestion
If you perform a DNS update that changes the endpoints that Mule runtimes use to connect to Runtime Manager, you must restart the Mule runtimes before they can connect to the new endpoints.
```

Reply:
(I had to read it twice to parse, so added the "that".)

FWIW: Style guide says to use "Mule instances" rather than "Mule runtimes".